config/datanode.example.toml

```diff
@@ -19,6 +19,9 @@ enable_telemetry = true
 ## Parallelism of initializing regions.
 init_regions_parallelism = 16
 
+## The maximum current queries allowed to be executed. Zero means unlimited.
+max_concurrent_queries = 0
```

Comment:
Please also update the example in `standalone.example.toml`
https://github.com/GreptimeTeam/greptimedb/blob/5c64f0ce0950e653ac2238c61b37b32a489810c7/config/standalone.example.toml#L15-L17

Reply:
TYFR this is done

Reply:
@evenyag Please check if this comment is resolved so we can proceed.